### ISSUE TITLE
Doesn't work with Gradle plugins block

### ISSUE THREAD

**Issue by EchoEllet** [mini2Dx/gettext#14]
To migrate from legacy apply plugin to plugins block, the alternative of:

```groovy
buildscript {
    repositories {
	mavenLocal()
        mavenCentral()
    }
    dependencies {
        classpath 'org.mini2Dx:gettext-gradle-plugin:1.7.1'
    }
}

project(":your-project") {
    apply plugin: "org.mini2Dx.gettext"
...

```

would be:

```kotlin
plugins {
    id("org.mini2Dx.gettext") version "1.11.0"
}
```

But it didn't work for some reason, any idea?

I asked on Slack and a developer called `Björn Kautler` (@Vampire on Slack) said:
`it does not publish the marker artifacts that are necessary to translate from the Plugin ID to the actual code artifact.`

The reason I'm interested in using the plugins block is not only for migrating to the new way of doing things but also for adding support to Gradle Kotlin DSL, you don't have to write it in Kotlin DSL (.KTS), only the publishing bug needs to be fixed, he also said `Type-safe accessors are only generated for plugins you apply in the plugins { ... ] block`

Thank you.

**Comment by hakanai** [comment]
Please publish Gradle plugins to the Gradle plugin portal. It is the proper repository for such things.

That is, if you think this plugin is any good. Otherwise, there are 4 or 5 other gettest plugins people could choose from which probably all do the same thing.

**Comment by Vampire** [comment]
Where the plugin is published is not so relevant, only for discoverability. Relevant is only that the marker artifacts are published too. And this can be worked-around using a plugin resolution strategy in the settings script to mitigate the missing plugin marker artifacts.

**Comment by hakanai** [comment]
Yes indeed. I was hoping not to have to introduce such workarounds.
